### PR TITLE
advance usage in server

### PR DESCRIPTION
**Pull request by alfonsmartinez**
Get request arguments to advance usage in server

### CODE REVIEW

**Comment by ondrejbartas** [comment]
I added similar code to my fork and than I saw your PR 👍 

add this snippet to readme so it is more friendly for newcommers


| query param | type | default value |
| - | - | - |
| model_name | str | u2net |
| alpha_matting | bool | False |
| alpha_matting_foreground_threshold | int | 240 |
| alpha_matting_background_threshold | int | 10 |
| alpha_matting_erode_structure_size | int | 10 |


``
| query param | type | default value |
| - | - | - |
| model_name | str | u2net |
| alpha_matting | bool | False |
| alpha_matting_foreground_threshold | int | 240 |
| alpha_matting_background_threshold | int | 10 |
| alpha_matting_erode_structure_size | int | 10 |
```

File: src/rembg/cmd/server.py [hunk 1]
```diff
@@ -14,6 +14,10 @@
 @app.route("/", methods=["GET", "POST"])
 def index():
     file_content = ""
+    alpha_matting = request.args.get("alpha_matting", type=bool, default="False")
```
Comment (ondrejbartas) [inline src/rembg/cmd/server.py:17]: I am no python developer but shouldn't be `default=False` without quotes?

Comment (AltayAkkus) [reply src/rembg/cmd/server.py:17]: yes it should!

**Comment by avaleriani** [comment]
Any way I can help with this?

**Comment by danielgatis** [comment]
closed in favor of https://github.com/danielgatis/rembg/pull/56

**Pull request closed without merging**